.gitignore

```diff
@@ -1 +1,2 @@
 .DS_Store
+node_modules/
```

Comment:
General question - did you pull the [Covalent Quickstart](https://github.com/Teradata/covalent-quickstart) files into here or was it a different project?

Reply:
Yes, these are from QuickStart

Reply:
Yes, these are from QuickStart

Reply:
Ah okay, it was recently updated - this repo can be updated in a future PR to keep it the scope of it smaller.